src/os/bluestore/BlueStore.cc

```diff
@@ -777,6 +777,8 @@ BlueStore::BlueStore(CephContext *cct, const string& path)
     kv_stop(false),
     logger(NULL)
 {
+  zeros = buffer::create_page_aligned(1048576);
```

Comment:
Do we really need to consume entire megabyte for just that? I think 128K is more than enough.